libs/ui-components/src/utils/patch.ts

```diff
@@ -200,3 +206,54 @@ export const getApplicationPatches = (
 
   return patches;
 };
+
+export const getMicroShiftHookPatches = (
+  currentHooks: DeviceHooksSpec | undefined,
+  registerMicroShift: boolean,
+  basePath: string,
+) => {
+  const patches: PatchRequest = [];
+  if (registerMicroShift) {
+    if (!currentHooks) {
+      patches.push({
+        op: 'add',
+        path: `${basePath}/hooks/`,
```

Comment:
Seems to fail due to the trailing slash.

Reply:
fixed